commit 000ffb26ea3e87484bf6b2583bd2cd2c7040a513
Author: Ben Wolsieffer <benwolsieffer@gmail.com>
Date:   Sun Dec 9 16:46:00 2018 -0500

    dtoverlays: fe-pi-audio: fix sgtl5000 compatible string
    
    The compatible string was set to "fepi,sgtl5000", which worked for some
    reason in 4.14, but does not work in 4.19, presumably due to some
    change in the kernel matching logic. The correct string is
    "fsl,sgtl5000".
    
    Signed-off-by: Ben Wolsieffer <benwolsieffer@gmail.com>

diff --git a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
index 81a07ed5a8c7..81dfa5aabe55 100644
--- a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
@@ -39,7 +39,7 @@
 
 			sgtl5000@0a {
 				#sound-dai-cells = <0>;
-				compatible = "fepi,sgtl5000";
+				compatible = "fsl,sgtl5000";
 				reg = <0x0a>;
 				clocks = <&sgtl5000_mclk>;
 				micbias-resistor-k-ohms = <2>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1402_000ffb26ea3e87484bf6b2583bd2cd2c7040a513.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1402_000ffb26ea3e87484bf6b2583bd2cd2c7040a513.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
